web/src/main/java/com/microsoft/applicationinsights/web/internal/httputils/AIHttpServletListener.java

```diff
@@ -13,14 +13,24 @@
 import org.apache.commons.lang3.Validate;
 import org.apache.http.annotation.Experimental;
 
+/**
+ * This class implements {@link AsyncListener} to handle span completion for async request handling.
+ */
 @Experimental
 public final class AIHttpServletListener implements Closeable, AsyncListener {
 
+    /**
+     * Instance of {@link RequestTelemetryContext}
+     */
     private final RequestTelemetryContext context;
+
+    /**
+     * Instance of {@link HttpServerHandler}
+     */
     private final HttpServerHandler<HttpServletRequest, HttpServletResponse> handler;
 
     public AIHttpServletListener(HttpServerHandler<HttpServletRequest, HttpServletResponse> handler,
-                                RequestTelemetryContext context) {
+                                    RequestTelemetryContext context) {
```

Comment:
minor - indentation should align with HttpServerHandler?

Reply:
fixed